### DISCUSSION TITLE
Sequential imputation using mice

### DISCUSSION THREAD

**Discussion by JevithaLobo** [amices/mice#660] [Getting started]
Usually when we conduct sequential regression to impute missing values in repeated measures, the pattern of missing should be in monotone pattern. Is mice package take care this pattern if missingness is non-monotone ? while imputing using sequential regression?

**Comment by stefvanbuuren** [comment]
Yes, `mice` uses Fully Conditional Specification and iterates over the variables. There is no need to have a monotone pattern. Vice versa, if your data are missing in a monotone pattern, there is no need to iterate. 

See https://stefvanbuuren.name/fimd/sec-monotone.html for details.

**Reply by JevithaLobo** [reply]
Thank you for clarification.
